packages/slice-machine/server/src/api/state.ts

```diff
@@ -48,8 +48,8 @@ export async function createWarnings(
   ) as ReadonlyArray<Warning>;
 }
 
-export default async function handler() {
-  const { env, errors: configErrors } = await getEnv();
+export default async function handler(req: RequestWithEnv) {
+  const { errors: configErrors, env } = req;
```

Comment:
Most of this function has changed in the release v0.1.3 branch, so I think we will have synchronization to do in this part :p

Reply:
no problem :)